src/Frontend/Core/Engine/TwigTemplate.php

```diff
@@ -0,0 +1,536 @@
+        //$twig->addFilter(new \Twig_SimpleFilter('stripnewlines', 'Frontend\Core\Engine\TemplateModifiers::stripNewlines'));
+        $twig->addFilter(new \Twig_SimpleFilter('formatdate', 'Frontend\Core\Engine\TemplateModifiers::formatDate'));
+        //$twig->addFilter(new \Twig_SimpleFilter('formattime', 'Frontend\Core\Engine\TemplateModifiers::formatTime'));
+        $twig->addFilter(new \Twig_SimpleFilter('formatdatetime', 'Frontend\Core\Engine\TemplateModifiers::formatDateTime'));
+        //$twig->addFilter(new \Twig_SimpleFilter('formatnumber', 'Frontend\Core\Engine\TemplateModifiers::formatNumber'));
+        $twig->addFilter(new \Twig_SimpleFilter('tolabel', 'Frontend\Core\Engine\TemplateModifiers::toLabel'));
+        $twig->addFilter(new \Twig_SimpleFilter('timeago', 'Frontend\Core\Engine\TemplateModifiers::timeAgo'));
+        $twig->addFilter(new \Twig_SimpleFilter('cleanupplaintext', 'Frontend\Core\Engine\TemplateModifiers::cleanupPlainText'));
+    }
+
+    /**
+     * Transform the string to uppercase.
+     *
+     * @return  string          The string, completly uppercased.
+     * @param   string $string  The string that you want to apply this method on.
+     */
+    public static function uppercase($string)
+    {
+        return mb_convert_case($string, MB_CASE_UPPER, \Spoon::getCharset());
+    }
+
+    /**
+     * Formats a language specific date.
+     *
+     * @return  string                      The formatted date according to the timestamp, format and provided language.
+     * @param   mixed $timestamp            The timestamp or date that you want to apply the format to.
+     * @param   string[optional] $format    The optional format that you want to apply on the provided timestamp.
+     * @param   string[optional] $language  The optional language that you want this format in (Check SpoonLocale for the possible languages).
+     */
+    public static function spoonDate($timestamp, $format = 'Y-m-d H:i:s', $language = 'en')
+    {
+        if(is_string($timestamp) && !is_numeric($timestamp)) {
+            // use strptime if you want to restrict the input format
+            $timestamp = strtotime($timestamp);
+        }
+        return \SpoonDate::getDate($format, $timestamp, $language);
+    }
+
+    /** @todo Refactor out constants #1106
+     *
+     * We need to deprecate this asap
+    */
+    private function startGlobals(&$twig)
+    {
+        // some old globals
+        $twig->addGlobal('var', '');
+        $twig->addGlobal('CRLF', "\n");
+        $twig->addGlobal('TAB', "\t");
+        $twig->addGlobal('now', time());
+        $twig->addGlobal('LANGUAGE', FRONTEND_LANGUAGE);
+        $twig->addGlobal('is' . strtoupper(FRONTEND_LANGUAGE), true);
+        $twig->addGlobal('debug', $this->debugMode);
+
+        $twig->addGlobal('timestamp', time());
+        $twig->addGlobal('timeFormat', Model::getModuleSetting('Core', 'time_format'));
```

Comment:
the getModuleSetting calls should be refactored to `get('fork.settings')->get` calls (see https://github.com/forkcms/forkcms/pull/1196)